src/HIE/Bios/Cradle.hs

```diff
@@ -155,9 +155,20 @@ processCabalWrapperArgs :: String -> Maybe [String]
 processCabalWrapperArgs args =
     case lines args of
         [dir, ghc_args] ->
-            let final_args = removeInteractive $ map (fixImportDirs dir) (words ghc_args)
+            let final_args =
+                    removeInteractive
+                    $ map (fixImportDirs dir)
+                    $ limited ghc_args
             in Just final_args
         _ -> Nothing
+  where
+    limited :: String -> [String]
+    limited = unfoldr $ \argstr ->
```

Comment:
Why did you call this `limited`?

Reply:
I didn't think of a better name for the opposite of `delimited`. Of course, I have no attachment to the name and will gladly change it. :smiley: